tests/clients/test_securitydata.py

```diff
@@ -219,6 +218,76 @@
             "fileSHA256": "testsha256-3",
             "versionTimestamp": 12346
         }
+    ],
+    "securityEventVersionsMatchingChecksum": [],
+    "securityEventVersionsAtPath": []
+}"""
+
+XFC_EXACT_FILE_VERSIONS = """{
+    "preservationVersions": [],
+    "securityEventVersionsMatchingChecksum": [
+        {
+            "edsUrl": "https://host-1.com",
```

Comment:
should we do `"https://host-1.example.com"` so this never resolves on accident

Reply:
I do tend to recommend using `example.com` due to the rule that it can never be anything, but don't feel it's worth searching the test suite for stuff that isn't that just to make that switch. That being said, since I was touching this code anyway,  it's probably a good thing to do.

Reply:
In other spots, I ust meant `host-2` etc, in the same blob